src/bankid.ts

```diff
@@ -361,10 +370,42 @@ export class BankIdClient {
     const passphrase = this.options.passphrase;
 
     return axios.create({
+      baseURL,
       httpsAgent: new https.Agent({ pfx, passphrase, ca }),
       headers: {
         "Content-Type": "application/json",
       },
     });
   }
 }
+
+interface AuthOptionalRequirementsV6 {
+  pinCode: boolean;
+  cardReader?: "class1" | "class2";
+  mrtd: boolean;
+  certificatePolicies?: string[];
+  personalNumber: string;
+}
```

Comment:
This looks sufficiently similar to the one in V5, should we make a common base out of it?